Makefile

```diff
@@ -1,53 +1,25 @@
-BASEIMAGE ?= "alpine:3.14"
+ON_PLUGINS ?= flase
```

Comment:
flase?

Reply:
The default is false because the built-in storage layer is likely to be used most of the time and cgo compilation is much slower.

I thought about it again, support for the storge plugin should be our default behavior, indeed it should be set to `true`, and only support building cgo images.

```dockerfile
ARG BUILDER_IMAGE
ARG BASE_IMAGE

FROM --platform=$BUILDPLATFORM ${BUILDER_IMAGE} as builder
WORKDIR /clusterpedia

ARG BIN_NAME
ARG ON_PLUGINS
ARG TARGETARCH
RUN GOARCH=${TARGETARCH} ON_PLUGINS=${ON_PLUGINS} make ${BIN_NAME}

FROM ${BASE_IMAGE}

ARG ON_PLUGINS
RUN if [ "${ON_PLUGINS}" = "true" ]; then apk add --no-cache gcompat; fi

ARG BIN_NAME
COPY --from=builder /clusterpedia/bin/${BIN_NAME} /usr/local/bin/${BIN_NAME}
```
->
```dockerfile
ARG BUILDER_IMAGE
ARG BASE_IMAGE

FROM --platform=$BUILDPLATFORM ${BUILDER_IMAGE} as builder
WORKDIR /clusterpedia

ARG BIN_NAME
ARG TARGETARCH
RUN GOARCH=${TARGETARCH} make ${BIN_NAME}

FROM ${BASE_IMAGE}

RUN apk add --no-cache gcompat

ARG BIN_NAME
COPY --from=builder /clusterpedia/bin/${BIN_NAME} /usr/local/bin/${BIN_NAME}
```

Reply:
ON_PLUGINS ?= flase ===>ON_PLUGINS ?= true

Reply:
ON_PLUGINS ?= flase ===> ON_PLUGINS ?= true

Reply:
ok we can make this behavior default

Reply:
updated